### PR TITLE
Fix computer play handling

### PR DESCRIPTION
**Pull request by sveinng**
# What?

Minor fixes to player vs computer web play
# Why?

Incorrect flow in computer gameplay. Computer win was not detected and ugly hax to display player1 name after computer play.
# Reviewer checklist

[ ] - The change introduces no untested code
[ ] - The change passes all existing tests
[ ] - The change introduces no typos or formatting errors

### CODE REVIEW

File: src/main/java/is/ru/tictactoe/webui/WebUI.java [hunk 1]
```diff
@@ -96,26 +96,26 @@ public static ModelAndView rootGetHandler(Request request, Response response) {
...
         		do {
             		int cellNum = ran.nextInt(10);
                     int y = cellNum / game.getBoard().boardSize();
                     int x = cellNum % game.getBoard().boardSize();
                 	try {
 						game.makeMove(x, y, 2);
-						valid = true;
-						// Set player name for next round
-			        	playerName = player1.getName();
+			            response.redirect("/");
```
Comment (solvip) [inline src/main/java/is/ru/tictactoe/webui/WebUI.java]: Indentation looks off?

**Review by solvip — APPROVED** [review]
One nitty comment; otherwise LGTM!

File: src/main/java/is/ru/tictactoe/webui/WebUI.java (hunk 1 above)
Comment (sveinng) [reply src/main/java/is/ru/tictactoe/webui/WebUI.java]: yes - i know 👎
Getting tired and careless 🐙  (and tabs vs space hussle)

**Review by eirikura — APPROVED** [review]
LGTM 💯

**Pull request merged**